### DISCUSSION TITLE
No way to disable certain functionality

### DISCUSSION THREAD

**Discussion by johannesschaffer** [AlessioGr/payload-plugin-lexical#17] [Feature Requests]
- Code block
- Bold
- Italic
- Underline
- Lists (numbered, bullet, check)
- Quote
- Import & Export
- Read-only mode
- Convert from markdown
- Insert -> Upload
- Formatting -> Strikethrough, Subscript, Superscript

If I remove everything from `defaultEditorConfig.features` & set all `defaultEditorConfig.<toggle>.enabled = false`, I would expect to have basically an empty editor

**Comment by AlessioGr** [comment]
Converted it to a feature request discussion. Will definitely make sure all those can be configured as well!

**Comment by ssyberg** [comment]
Is this still true, there's no way to disable these options?

**Reply by AlessioGr** [reply]
Most of these still cannot be disabled. I am currently focusing on lexical in the core rather than the lexical plugin and, other than a converter, won't be making any new changes in this plugin.

Lexical-core will be **a lot** more customizable!

**Comment by ssyberg** [comment]
> Lexical-core will be a lot more customizable!

That's great to hear!
